### PR TITLE
Migrated from hugo-geekdoc to ps-docs-theme

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Migrated from hugo-geekdoc to ps-docs-theme
| Fixed ticket? | N/A

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -6,17 +6,46 @@ canonifyURLs: true
 relativeURLs: true
 uglyURLs: true
 
-theme: "hugo-geekdoc"
+theme: "ps-docs-theme"
+enableGitInfo: true
+enableRobotsTXT: true
+googleAnalytics: "UA-2753771-44"
```
Comment (eternoendless) [inline config.yml:12]: This has to be removed until we get a GA key for specdocs

File: config.yml [hunk 2]
```diff
@@ -6,17 +6,46 @@ canonifyURLs: true
...
-  geekdocLogo: "logo.png"
-  geekdocRepo: "https://github.com/PrestaShop/prestashop-specs"
-  geekdocSearch: true
-  geekdocSearchShowParent: true
-  geekdocMenuBundle: false
+  editURL: "https://github.com/PrestaShop/prestashop-specs/edit/master/content/"
+  ghRepoURL: "https://github.com/PrestaShop/prestashop-specs/"
+  ghCommitURL: "https://github.com/PrestaShop/prestashop-specs/commit/"
+  howToContributeURL: "https://devdocs.prestashop.com/1.7/contribute/documentation/how/"
+  description: "Learn how to extend, modify and test PrestaShop, create modules, themes, and more."
```
Comment (eternoendless) [inline config.yml:19]: Needs to be changed

File: content/_index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Home
+---
+
+# PrestaShop Specs Documentation
+
+PrestaShop is a free and Open Source e-commerce web platform, committed to providing the best shopping cart experience for both merchants and customers. It is written in PHP, is highly customizable, supports all the major payment services, is translated in many languages and localized for many countries, has a fully responsive design (both front and back office), and more! [Learn more about PrestaShop](https://www.prestashop.com/). 
+
+This site covers the specifications of the project PrestaShop.
```
Comment (eternoendless) [inline content/_index.md:9]:
```suggestion
This site covers the specifications of the PrestaShop project.
```

File: content/_index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Home
+---
+
+# PrestaShop Specs Documentation
```
Comment (eternoendless) [inline content/_index.md:5]:
```suggestion
# PrestaShop Specifications
```

File: config.yml [hunk 5]
```diff
@@ -6,17 +6,46 @@ canonifyURLs: true
...
+  author: "PrestaShop"
+  showVisitedLinks: false
+  disableInlineCopyToClipBoard: true
+  projectUrls:
+    gitHub: "https://github.com/PrestaShop/PrestaShop/"
+    reportBug: "https://github.com/PrestaShop/PrestaShop/issues"
+    slack: "https://www.prestashop-project.org/slack/readme/"
+  coverImage: "images/social-cover.jpg"
```
Comment (eternoendless) [inline config.yml:27]: Where's this image?

**Review by eternoendless — COMMENTED** [review]
Can you please provide a rendered image of what it looks like?

**Comment by Progi1984** [comment]
@eternoendless I fixed your feedback.

The screenshot : 
image

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @Progi1984

**Pull request merged**